src/copy.rs

```diff
@@ -752,7 +752,7 @@ fn prepare_copy_internal(options: Options,
                               "UTF8_STRING",
                               "TEXT"];
             for &mime_type in &text_mimes {
-                // We don't want to overwrite an explicit mime type, because it might be bound to a
+                // We don't want to overwrite an explicit mime type, beerror(source) it might be bound to a
```

Comment:
uh oh :)

Reply:
The perils of string replacement :joy:

Actually, there were multiple of it